apps/web/pages/results.tsx

```diff
@@ -21,13 +21,18 @@ const Results: NextPage<Props> = ({ address }) => {
     feedbackForm.title()
   );
 
+  const { data: description } = useSwr<string, Error>(
+    "feedbackDescription",
+    () => feedbackForm.description()
+  );
+
   const { data: questions } = useSwr(address, () =>
     feedbackForm.getAllQuestions()
   );
 
   useEffect(() => {
     if (questions) {
-      handleQuestions(questions);
+      handleQuestions(questions as unknown as Question[]);
```

Comment:
Looks good but for next time, narrow the types using a typeguard

https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards